doc/Language/system.pod6

```diff
@@ -84,7 +84,7 @@ L<C<P5getprotobyname>|https://github.com/lizmat/P5getprotobyname>, which
 includes functions such as C<endprotoent>, C<getprotoent>, C<getprotobyname>,
 C<getprotobynumber> and C<setprotoent>.
 L<Search and install C<P5> modules|https://modules.raku.org/search/?q=p5>
-if you want to use those functions already in p6y form.
+if you want to use those functions already in Raku-y form.
```

Comment:
I think I've heard “Rakuish” before? Not sure what's better :)

Reply:
"Rakuish" is better, yes.